### PR TITLE
Add github-git-cheat-sheet in zh_TW

### PR DESCRIPTION
**Pull request by extend1994**
## Overview
**TL;DR**
Add github-git-cheat-sheet.md in zh_TW and update index.html accordingly.

### Questions
No Questions :)

### Next Steps
Wait for reviewers' opinions to decide the next steps. 

### Review
Hi @parkerbxyz @brianamarie, I noticed that you were handling the recent pull requests, so I tagged you here hoping that you can help review this PR. Hope that it doesn't bother you.

### CODE REVIEW

**Review by parkerbxyz — APPROVED** [review]
Thanks, @extend1994! We usually like to get a second review from another native speaker before merging. Do you know anybody that could provide such a review?

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:10]: It is subtle, but the "装" here is actually Simplified Chinese. You may use "裝" instead.

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:51]: "删" seems to be Simplified Chinese too. You may use "刪" instead.

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:77]: It means ".gitignore file" which should be ".gitignore 檔".

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:83]: Please beware of consistency, so it may be "本機端".

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:91]: Same above.

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:95]: Same as above.

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:99]: Same as above.

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:131]: Should be 檔案

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:143]: I believe "revert" is usually translated to "復原", but it is up to you.

Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:143]: And "本機" consistency

File: downloads/zh_TW/github-git-cheat-sheet.md
Comment (ralphchung) [inline downloads/zh_TW/github-git-cheat-sheet.md:149]: "Contact support" should be "尋求支援" or something similar. No matter what, the word itself should be "聯絡" or "聯繫".

**Comment by parkerbxyz** [comment]
Thanks for the review, @ralphchung! Would you be able to open a pull request with your suggested changes?

**Comment by extend1994** [comment]
Hi @parkerbxyz, I opened #937 for the above suggestions, feel free to review it 😃

**Pull request merged**
